src/python/turicreate/test/test_style_transfer.py

```diff
@@ -125,6 +126,53 @@ def test_create_with_incorrect_max_iterations_format_float(self):
         with self.assertRaises(_ToolkitError):
             tc.style_transfer.create(self.style_sf[:1], self.content_sf[:1], max_iterations=1.25)
 
+    def test_create_with_verbose_False(self):
+        # Train a model with verbose=False
+        old_stdout = sys.stdout
+        sys.stdout = stdout_without_verbose = _StringIO()
+        model = tc.style_transfer.create(self.style_sf,
+                                              self.content_sf,
+                                              style_feature=self.style_feature,
+                                              content_feature=self.content_feature,
+                                              max_iterations=1,
+                                              model=self.pre_trained_model,
+                                              verbose=False)
+        sys.stdout = old_stdout
+        without_verbose = stdout_without_verbose.getvalue()
+        # Train a model with verbose=True
+        old_stdout = sys.stdout
+        sys.stdout = stdout_with_verbose = _StringIO()
+        model = tc.style_transfer.create(self.style_sf,
+                                              self.content_sf,
+                                              style_feature=self.style_feature,
+                                              content_feature=self.content_feature,
+                                              max_iterations=1,
+                                              model=self.pre_trained_model,
+                                              verbose=True)
+        sys.stdout = old_stdout
+        with_verbose = stdout_with_verbose.getvalue()
+        # Assert that verbose logs are longer
+        assert (len(with_verbose) > len(without_verbose))
+
+    def test_stylize_with_verbose_False(self):
+        sf = self.content_sf[0:1]
```

Comment:
I'm very impressed with these unit tests. However the fact that you've got very similar code in a few different places makes me think you should probably move most of that code to a helper function. The right place for that helper function is probably: `src/python/turicreate/test/util.py`.

Reply:
Done!